src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.data.person;
+import seedu.addressbook.data.exception.*;
+public class Contact {
+    public String value;
```

Comment:
try to make it private. in general start off from private and use protected/public only if absolutely necessary